src/libs/ReportUtils.js

```diff
@@ -1838,6 +1839,7 @@ function getIOUReportActionMessage(type, total, comment, currency, paymentType =
             iouMessage = `deleted the ${amount} request${comment && ` for ${comment}`}`;
             break;
         case CONST.IOU.REPORT_ACTION_TYPE.PAY:
+            amount = CurrencyUtils.convertToDisplayString(getMoneyRequestTotal(getReport(iouReportID)), currency);
```

Comment:
We cannot use it for all types because for CREATE for example we only care about the request amount but not about the total of the report (can be bunch of the requests)